zilliqa/src/consensus.rs

```diff
@@ -2438,6 +2492,14 @@ impl Consensus {
         self.view.get_view()
     }
 
+    /// Set view in memory and update tip_info.latest_view in storage
```

Comment:
Do we need to keep view in memory at all? I'd prefer a single source-of-truth unless there's a good reason not to?